README.md

```diff
@@ -245,6 +247,14 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Compatibility with other annotation processors
+
+The plugin should be compatible with other annotation processors, as long
```

Comment:
"be compatible with" is ambiguous.  Be more specific.  You could cut the first sentence, because the second sentence starts with the important information.